README.md

```diff
@@ -1,161 +1,27 @@
-# ArduPilot Project
+# Altitude Control for Waypoints Using Throttle Input
```

Comment:
changes to this file are totally inappropriate...pleas delete all changes.
If and when a new feature is added, it could be announced via a blog post, if judged noteworthy enough and approved, in discuss.ardupilot.org